charts/newrelic-infrastructure/templates/kubelet/daemonset-windows.yaml

```diff
@@ -0,0 +1,70 @@
+{{- if and .Values.kubelet.enabled .Values.kubelet.enableWindows }}
+{{- range .Values.kubelet.windowsOsList }}
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  namespace: {{ $.Release.Namespace }}
+  name: {{ include "nriKubernetes.kubelet.windows.fullname" $ }}-{{ .version }}
+  {{- $legacyAnnotation:= fromYaml (include "newrelic.compatibility.annotations" $) -}}
+  {{- with  include "newrelic.compatibility.valueWithFallback" (dict "legacy" $legacyAnnotation "supported" $.Values.kubelet.annotations )}}
+  annotations: {{ . | nindent 4 }}
+  {{- end }}
+spec:
+  {{- with $.Values.updateStrategy }}
+  updateStrategy: {{ toYaml . | nindent 4 }}
+  {{- end }}
+  selector:
+    matchLabels:
+      {{- include "newrelic.common.labels.selectorLabels" $ | nindent 6 }}
+      app.kubernetes.io/component: kubelet
+  template:
+    metadata:
+      annotations:
+        checksum/nri-kubernetes: {{ include (print $.Template.BasePath "/kubelet/scraper-configmap.yaml") $ | sha256sum }}
```

Comment:
Nri-kubernetes in V2 branch does not read any configuration, that was introduced in V3.

I wonder if you actually compiled the V3 branch and pushed it to the old repo `newrelic/infrastructure-k8s`, is that the case?

There are some discrepancies:
 - kubelet/scraper-configmap.yaml is V3
 - NRIA_PASSTHROUGH_ENVIRONMENT is V2
 - app.kubernetes.io/component: kubelet is V3
 - having just one container is definitely V2

Reply:
how a user could change the config mounted? or set a proxy?